docs/examples/numba_func/Numba_func.ipynb

```diff
@@ -0,0 +1,370 @@
+{
```

Comment:
<p>"Since the output shape will be different than the input, we need to implement a setup function as well."</p><p>"In the setup function, we are defining the output data shapes in terms of the input shapes."</p><p> </p><p>"If the setup function is not provided, the operator assumes that the types and shapes of the outputs are the same as the in the inputs." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>"If the setup function is not provided, the operator assumes that the the outputs are of the same shape as the inputs."</p><p> </p><p>We don't set type at run-time.</p>